pina/label_tensor.py

```diff
@@ -66,7 +66,36 @@ def __init__(self, x, labels):
                 'the tensor has not the same number of columns of '
                 'the passed labels.'
             )
-        self.labels = labels
+        self._labels = labels
+
+    def _exist_labels(self):
+        """Check the existence of labels in a label tensor
+
+        :return: Returns True if the tensor has already
+        a label, False otherwise
```

Comment:
one indentation is missing

Reply:
thank you for the reply, a more compact form is provided